t/tikz/ac-drive-voltage.xml

```diff
@@ -0,0 +1,119 @@
{}{{}}}{{}{}}{}{{}{}}&#10;{&#10;}{{{{}}\pgfsys@beginscope\pgfsys@invoke{ }\pgfsys@transformcm{1.0}{0.0}{0.0}{1%&#10;.0}{-28.92323pt}{111.3666pt}\pgfsys@invoke{ }\hbox{{\definecolor{%&#10;pgfstrokecolor}{rgb}{0,0,0}\pgfsys@color@rgb@stroke{0}{0}{0}\pgfsys@invoke{ }%&#10;\pgfsys@color@rgb@fill{0}{0}{0}\pgfsys@invoke{ }\hbox{{$U_{s},\varPsi_{s}$}}&#10;}}\pgfsys@invoke{\lxSVG@closescope }\pgfsys@endscope}}}&#10;\pgfsys@invoke{\lxSVG@closescope }\pgfsys@endscope}}}&#10;{}{{}}{}&#10;{}{}\pgfsys@beginscope\pgfsys@invoke{ }\pgfsys@setdash{0.4pt,2.0pt}{0.0pt}%&#10;\pgfsys@invoke{ }{}\pgfsys@moveto{56.90552pt}{0.0pt}\pgfsys@lineto{56.90552pt}%&#10;{113.81104pt}\pgfsys@stroke\pgfsys@invoke{ }&#10;\pgfsys@invoke{\lxSVG@closescope }\pgfsys@endscope\par{}{{}}{}&#10;{}{}&#10;{}{}\pgfsys@beginscope\pgfsys@invoke{ }\pgfsys@setlinewidth{0.8pt}%&#10;\pgfsys@invoke{ }{}\pgfsys@moveto{0.0pt}{0.0pt}\pgfsys@lineto{56.90552pt}{56.9%&#10;0552pt}\pgfsys@lineto{170.71655pt}{56.90552pt}\pgfsys@stroke\pgfsys@invoke{ }&#10;\pgfsys@invoke{\lxSVG@closescope }\pgfsys@endscope{}{{}}{}{{}}{}\pgfsys@moveto%&#10;{28.45276pt}{42.67914pt}\pgfsys@stroke\pgfsys@invoke{ }\hbox{\hbox{{%&#10;\pgfsys@beginscope\pgfsys@invoke{ }{{}{}{{&#10;{}{}}}{&#10;{}{}}&#10;{{}{{}}}{{}{}}{}{{}{}}&#10;{&#10;}{{{{}}\pgfsys@beginscope\pgfsys@invoke{ }\pgfsys@transformcm{1.0}{0.0}{0.0}{1%&#10;.0}{23.18124pt}{39.86525pt}\pgfsys@invoke{ }\hbox{{\definecolor{pgfstrokecolor%&#10;}{rgb}{0,0,0}\pgfsys@color@rgb@stroke{0}{0}{0}\pgfsys@invoke{ }%&#10;\pgfsys@color@rgb@fill{0}{0}{0}\pgfsys@invoke{ }\hbox{{$U_{s}$}}&#10;}}\pgfsys@invoke{\lxSVG@closescope }\pgfsys@endscope}}}&#10;\pgfsys@invoke{\lxSVG@closescope }\pgfsys@endscope}}}&#10;\par{}{{}}{}&#10;{}{}{}{{}{}}{{}{}{{{}{}{}}}}\pgfsys@beginscope\pgfsys@invoke{ }%&#10;\pgfsys@setlinewidth{0.8pt}\pgfsys@invoke{ }\pgfsys@setdash{3.0pt,3.0pt}{0.0pt%&#10;}\pgfsys@invoke{ }{}\pgfsys@moveto{0.0pt}{85.35828pt}\pgfsys@lineto{56.90552pt%&#10;}{85.35828pt}\pgfsys@curveto{69.7096pt}{72.5542pt}{113.81104pt}{28.45276pt}{17%&#10;0.71655pt}{28.45276pt}\pgfsys@stroke\pgfsys@invoke{ }&#10;\pgfsys@invoke{\lxSVG@closescope }\pgfsys@endscope{}{{}}{}{{}}{}\pgfsys@moveto%&#10;{71.1319pt}{85.35828pt}\pgfsys@stroke\pgfsys@invoke{ }\hbox{\hbox{{%&#10;\pgfsys@beginscope\pgfsys@invoke{ }{{}{}{{&#10;{}{}}}{&#10;{}{}}&#10;{{}{{}}}{{}{}}{}{{}{}}&#10;{&#10;}{{{{}}\pgfsys@beginscope\pgfsys@invoke{ }\pgfsys@transformcm{1.0}{0.0}{0.0}{1%&#10;.0}{65.9305pt}{82.54439pt}\pgfsys@invoke{ }\hbox{{\definecolor{pgfstrokecolor}%&#10;{rgb}{0,0,0}\pgfsys@color@rgb@stroke{0}{0}{0}\pgfsys@invoke{ }%&#10;\pgfsys@color@rgb@fill{0}{0}{0}\pgfsys@invoke{ }\hbox{{$\varPsi_{s}$}}&#10;}}\pgfsys@invoke{\lxSVG@closescope }\pgfsys@endscope}}}&#10;\pgfsys@invoke{\lxSVG@closescope }\pgfsys@endscope}}}&#10;\par&#10;\pgfsys@invoke{\lxSVG@closescope }\pgfsys@endscope{}{}{}\hss}%&#10;\pgfsys@discardpath\pgfsys@invoke{\lxSVG@closescope }\pgfsys@endscope\hss}}%&#10;\lxSVG@closescope\endpgfpicture}}" width="303.8" xml:id="p1.pic1">
```

Comment:
While studying the XML in the tests here, I have been wondering if we should add a switch to disable the `tex` attribute in the `<picture>` element, especially when they come from some pgf/tikz source.

The expanded low-level pgf instructions get overwhelming quite quickly, and contain LaTeXML-only macros (such as `\lxSVG@close`) so they won't be reusable in other, non-latexml, contexts.

But even if we keep them in general, I can imagine various spurious diffs in these tests later on, e.g. if we refactor internal macro names.

Reply:
Ah, interesting: all the more so since the exact source snippets differ between texlives. I generated the tests on texlive 2022, and our 2018-2021 CI versions all failed on the source line.